### ISSUE TITLE
docs: readthedocs has stopped updating

### ISSUE THREAD

**Issue by jacquerie** [inspirehep/inspire-next#2386]
As I am sure everyone has noticed, because everyone frequently refers to the documentation of our project, since https://github.com/inspirehep/inspire-next/pull/2246 the documentation on http://inspirehep.readthedocs.io/en/latest/ has stopped updating.

**Comment by kaplun** [comment]
Thanks for pointing it out. I haven't noted it. I think this quite important to be fixed.

**Comment by jacquerie** [comment]
Still not fixed: http://inspirehep.readthedocs.io/en/latest/

**Comment by jacquerie** [comment]
@david-caro, @kaplun: It would be helpful to share what led you to believe that the fix in #2432 was correct, or in general how to diagnose doc building issues on ReadTheDocs, either locally or remotely.

**Comment by kaplun** [comment]
According to readthedocs.io the latest build passed: https://readthedocs.org/projects/inspirehep/builds/5556954/ 
You showed me IRL that there are some section missing (in particular autogenerated APIs). Maybe that we are missing some further extension on readthedocs, but that don't cause a failure in the build.

**Comment by jacquerie** [comment]
You're right that this unblocks https://github.com/inspirehep/inspire-next/pull/1944, https://github.com/inspirehep/inspire-next/pull/1950, and https://github.com/inspirehep/inspire-next/pull/2027, but not having the API documentation is still a problem: we can't ask people to document their code and then not have it show up anywhere...

**Comment by kaplun** [comment]
If I build locally with `python setup.py build_sphinx` I obtain this warning:
```
/home/skaplun/.virtualenvs/labs3/src/inspire-next/docs/index.rst:29: WARNING: toctree contains reference to nonexisting document u'_source/inspirehep'
```

Which seems to correspond to the missing part on readthedocs.org. 
What are you doing to compile locally?

**Comment by kaplun** [comment]
Ha! Instead:
```
cd docs
make html
```
does indeed load all the inspirehep source code.

**Comment by david-caro** [comment]
I think though that that issue should be solved in a different pr. And also another pr is needed to fix all the warning and set the 'warnings as errors' flag.

**Comment by jacquerie** [comment]
This was fixed by https://github.com/inspirehep/inspire-next/pull/2432 and https://github.com/inspirehep/inspire-next/pull/2439.
